### DISCUSSION TITLE
Behaviour of XF86Sleep Key

### DISCUSSION THREAD

**Discussion by NorbertJh** [lxqt/lxqt#2343] [Q&A]
With default key settings in Power Management and Global Actions Manager (Shortcuts) pressing the XF86Sleep key causes a shutdown.
Changing the Power Key Action in Power Management shows that pressing the XF86Sleep Key generally executes the Power Key Action.

I got around that effect by setting the Suspend Key Action in Power Management to Nothing and change the Global Action for XF86Sleep to “systemctl suspend” or “lxqt-leave --suspend”.

The computer is a Tuxedo Infinity Book S15 Gen6, the Linux is an up-to-date Arch Linux
(uname -a --> Linux jhn3 6.0.10-arch2-1 #1 SMP PREEMPT_DYNAMIC Sat, 26 Nov 2022 16:51:18 +0000 x86_64 GNU/Linux)
the LXQT version is 1.20 and the Qt version is 5.15.7.

**Comment by git-developer** [comment]
I'm experiencing the same behavior, and your workaround fixes the issue. Thanks for sharing!

IMHO this is a bug and the related issue #2342 should be re-opened. /cc @tsujan
